### PR TITLE
Removed "base64-encoded" from note 3 of step 1 of "Migrating VM" procedure

### PR DESCRIPTION
**Pull request by RichardHoch**
Fixes error in https://github.com/konveyor/forklift-documentation/pull/315. 

Removes "base64-encoded" from note 3 of step 1 of "Migrating VM" procedure

Preview: https://deploy-preview-316--forklift-documentation.netlify.app/documentation/doc-migration_toolkit_for_virtualization/master/#migrating-virtual-machines-cli_forklift

### CODE REVIEW

File: documentation/modules/migrating-virtual-machines-cli.adoc [hunk 1]
```diff
@@ -43,7 +43,7 @@ EOF
 ----
 <1> Specify the vCenter user or the {rhv-short} {manager} user.
 <2> Specify the user password.
-<3> {rhv-short} only: Specify the base64-encoded CA certificate of the {manager}. You can retrieve it at `https://<{rhv-short}_engine_host>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA`.
+<3> {rhv-short} only: Specify the CA certificate of the {manager}. You can retrieve it at `https://<{rhv-short}_engine_host>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA`.
 <4> VMware only: Specify the vCenter SHA-1 fingerprint.
```
Comment (RichardHoch) [inline documentation/modules/migrating-virtual-machines-cli.adoc:47]: @mguetta1 -- I just noticed that for upstream, {rhv-short} is not going to render as RHV.  I suggest writing it as RHV, without the attribute. What do you think?

Comment (mguetta1) [reply documentation/modules/migrating-virtual-machines-cli.adoc:47]: @RichardHoch we need the attribute, the RHV upstream project is oVirt.
I think it worked before, I remember that in the previews - {rhv-short} is oVirt

Comment (RichardHoch) [reply documentation/modules/migrating-virtual-machines-cli.adoc:47]: Great, then please review as is.

Comment (mguetta1) [reply documentation/modules/migrating-virtual-machines-cli.adoc:47]: Regarding the 'Ansible playbook' in the Hook manifest, I understood from @istein1 that it should be Base64-encoded.
Thanks

**Review by mguetta1 — APPROVED** [review]
Looks good to me

**Pull request merged**